app/src/main/java/com/android/shelfLife/ui/overview/AddFoodItem.kt

```diff
@@ -244,9 +334,11 @@ fun AddFoodItemScreen(
                         val foodFacts =
                             FoodFacts(
                                 name = foodName,
-                                barcode = "",
+                                barcode = selectedImage?.barcode ?: "",
                                 quantity = Quantity(amount.toDouble(), unit),
-                                category = category)
+                                category = category,
+                                nutritionFacts = selectedImage?.nutritionFacts ?: NutritionFacts(),
+                                imageUrl = selectedImage?.imageUrl ?: FoodFacts.DEFAULT_IMAGE_URL)
```

Comment:
## Auto complete feature possibility?
Since we save the selectedImage FoodFact, we could also give the user an autofill feature:
 - the user would type a name of a prduct
 - the user selects the right product
 - it would auto fill the quantity from the product data
 - it would auto fill expiry date by guessing from product data

This would expand on this current feature, and make a really amazing app-feature imo

Reply:
This was something i also realized while developing this feature, however I must be clear that we must keep in mind that if we are too depend on the FoodFacts API we are going to have issues when it comes to developing the offline mode of the app. Hence at the moment I am choosing to keep them seperate. On the other hand, i am planning a major redesing to the overal foodItem experience so watch out!